src/iguana/algorithms/clas12/FiducialFilter/Algorithm.h

```diff
@@ -0,0 +1,373 @@
+#pragma once
+
+#include "iguana/algorithms/Algorithm.h"
+#include "iguana/algorithms/TypeDefs.h"
+
+namespace iguana::clas12 {
+
+  /// @brief_algo Filter the `REC::Particle` bank by applying DC (drift chamber) fiducial cuts
+  ///
+  /// @begin_doc_algo{Filter}
```

Comment:
new docstring syntax (#223)
```suggestion
  /// @begin_doc_algo{clas12::FiducialFilter | Filter}
```